crates/client/storage/src/overrides/schema_v1_override.rs

```diff
@@ -58,12 +63,11 @@ where
     C: HeaderBackend<B> + StorageProvider<B, BE> + 'static,
     BE: Backend<B> + 'static,
 {
-    fn current_block(&self, block_hash: B::Hash) -> Option<StarknetBlock> {
-        self.query_storage::<StarknetBlock>(
-            block_hash,
-            &StorageKey(storage_prefix_build(PALLET_STARKNET, STARKNET_CURRENT_BLOCK)),
-        )
-        .map(Into::into)
+    fn current_block(&self, block_hash: <B as BlockT>::Hash) -> Option<StarknetBlock> {
```

Comment:
It has no reason to be in the StorageOverride trait.
We are not reading the storage, we are not using override.
In think you can just a convenient, well named, wrapper function in a `utils.rs` file in rpc